metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -62,11 +62,18 @@ def trim_query(self, query: str, budget_per_query: int) -> str:
                 )
         return trimmed_query
 
+    def indent_query(self, query: str, indent) -> str:
```

Comment:
Make this either a `@staticmethod` or use `self.indent` instead of passing `indent` to this method.

Reply:
Thanks for your advice: I made it a `@staticmethod`.